src/main/resources/versions.json

```diff
@@ -1,9 +1,9 @@
 {
     "supportedVersions": {
-        "8.1.0-alpha09": [
+        "8.1.0-alpha11": [
             "8.0.2"
```

Comment:
8.1 produces a regression in the Configuration Cache test where the inputs to the configuraiton cache test change. 
Keeping as 8.0.2 for now.

Here is a scan link to the regression: https://ge.solutions-team.gradle.com/s/okdyga6gfyuog/tests/:testAndroid8_0_0/org.gradle.android.ConfigurationCachingTest/plugin%20is%20compatible%20with%20configuration%20cache?focused-exception-line=0-6&top-execution=1